pydantic_ai_slim/pydantic_ai/run.py

```diff
@@ -135,6 +135,20 @@ def result(self) -> AgentRunResult[OutputDataT] | None:
             self._traceparent(required=False),
         )
 
+    def all_messages(self) -> list[_messages.ModelMessage]:
```

Comment:
Let's add the JSON versions as well, and clarify "for the run so far" as we have on the `usage` method.